Test/CLI_test.py

```diff
@@ -0,0 +1,158 @@
+import sys
+# look in ../ BEFORE trying to import Algorithmia.  If you append to the
+# you will load the version installed on the computer.
+sys.path = ['../'] + sys.path
+
+import unittest
+import os
+import Algorithmia
+from Algorithmia.CLI import CLI
+import argparse
+
+class CLITest(unittest.TestCase):
+	def setUp(self):
+		# create a directory to use in testing the cp command 
+		self.client = Algorithmia.client('simdylfCeXZ8/MgaQzokUHlalWm1')
+		CLI().mkdir("data://.my/moredata", self.client)
+	
+	def test_ls(self):
+		parentDir = "data://.my/"
+		newDir = "test"
+
+		CLI().mkdir(parentDir+newDir, self.client)
+		result = CLI().ls(parentDir, self.client)
+		self.assertTrue(result is not None and "moredata" in result and newDir in result)
+		
+		CLI().rmdir(parentDir+newDir, self.client)
+
+
+	def test_mkdir(self):
+
+		parentDir = "data://.my/"
+		newDir = "test"
+
+		CLI().mkdir(parentDir+newDir, self.client)
+		result = CLI().ls(parentDir, self.client)
+		self.assertTrue(newDir in result)
+		
+		CLI().rmdir(parentDir+newDir, self.client)
+
+	def test_rmdir(self):
+		parentDir = "data://.my/"
+		newDir = "testRmdir"
+
+		CLI().mkdir(parentDir+newDir, self.client)
+		result = CLI().ls(parentDir, self.client)
+		self.assertTrue(newDir in result)
+		
+		CLI().rmdir(parentDir+newDir, self.client)
+
+		result = CLI().ls(parentDir, self.client)
+		self.assertTrue(newDir not in result)
+
+	def test_cat(self):
+		file = "data://.my/moredata/test.txt"
+		fileContents = "some text in test file"
+
+		CLI().rm(file, self.client)
+		testfile = open("./test.txt", "w")
+		testfile.write(fileContents)
+		testfile.close()
+
+		CLI().cp(["./test.txt"],file,self.client)
+
+		result = CLI().cat([file],self.client)
+		self.assertEqual(result, fileContents)
+
+
+#local to remote
+	def test_cp_L2R(self):
+
+		testfile = open("./test.txt", "w")
+		testfile.write("some text")
+		testfile.close()
+
+		src = ["./test.txt"]
```

Comment:
Recommend changing to something like "./testdata/test.txt" and then add "testdata" to the gitignore file please.